### DISCUSSION TITLE
Re-publish scene

### DISCUSSION THREAD

**Discussion by collaborative-web** [mozilla/hubs#4327] [Spoke Questions]
Hi, I am trying to republish a scene to Hubs and maintain the same room name and link. When I published the updated scene, it created a new link and room name and I want to avoid this. Thanks in advance for your input!

**Comment by j-conrad** [comment]
As long as you do NOT change the name of the scene, re-publishing should maintain the same address for the scene. This means you could be in a Hubs room using that scene and simply hit 'Refresh' on your browser after re-publishing from Spoke and you should see the changes.
